xtask/src/fuzz.rs

```diff
@@ -1,5 +1,5 @@
 use crate::prelude::*;
-
+use crate::windows_skip;
```

Comment:
suggestion: Re-export the xtask macros in the prelude, I think this aligns better with the initial intention.